src/HotChocolate/AspNetCore/src/AspNetCore/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.AspNetCore.Builder;
 /// </summary>
 public static class EndpointRouteBuilderExtensions
 {
+    private const string GraphQLHttpPath = "/graphql";
+    private const string GraphQLWebSocketPath = "/graphql/ws";
+    private const string GraphQLSchemaPath = "/graphql/sdl";
+    private const string GraphQLToolPath = "/graphql/ui";
+    private const string GraphQLToolRelativeRequestPath = "..";
```

Comment:
The default needs to be "../graphql"

Reply:
For the regular ToolOptions yes, but this is for MapBananaCakePop, which resides at `/graphql/ui`, so `..` is correct.

Reply:
a you are right.